certbot-apache/certbot_apache/tests/parser_test.py

```diff
@@ -30,6 +43,19 @@ def test_aug_version(self):
         self.parser.aug.match.side_effect = RuntimeError
         self.assertFalse(self.parser.check_aug_version())
 
+    def test_recovery_routine_reload(self):
+        mock_load = mock.Mock()
+        self.config.parser.aug.load = mock_load
+        self.config.recovery_routine()
+        self.assertEqual(mock_load.call_count, 1)
+
+    @mock.patch("certbot_apache.parser.ApacheParser.init_augeas")
+    def test_prepare_no_augeas(self, mock_init_augeas):
```

Comment:
Can we move this to `ParserInitTest` and test that `errors.NoInstallationError` is raised out of the parser's `__init__` method instead of calling back into the configurator?